packages/nuxt/src/app/plugins/router.ts

```diff
@@ -246,7 +246,10 @@ export default defineNuxtPlugin<{ route: Route, router: Router }>({
               if (result === false || result instanceof Error) {
                 const error = result || createError({
                   statusCode: 404,
-                  statusMessage: `Page Not Found: ${initialURL}`
+                  statusMessage: `Page Not Found: ${initialURL}`,
+                  data: {
+                    to
```

Comment:
I think we should likely include just `to.fullPath` (maybe `path: to.fullPath`?) as otherwise the entire route object will (try to) be serialised on an SSR error, which I imagine will throw an error.

Reply:
I was thinking about that too first but thought including more info wouldn't hurt (or maybe excluding meta) 🤔

But only `fullPath` sounds good to me!